### PR TITLE
bug fix: input checkbox should be visible at hover

### PR DESCRIPTION
**Pull request by SMahil**
bug fix: input checkbox should only be visible at hover

#684 

Screenshot:

### CODE REVIEW

**Comment by debloper** [comment]
How to select multiple WI after this patch?
The moment I select a new WI, the old one gets deselected.

**Comment by SMahil** [comment]
@debloper I am not sure if we are focusing multiple selection

**Comment by nimishamukherjee** [comment]
@debloper the current implementation is for a single work item. We will have a different set of stories for multiple selection.

File: src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
   align-items: center;
   .row-cbh {
     margin-right: em(10);
     margin-left: em(5);
-    padding-right: em(10);
-    border-right: 1px solid $color-pf-black-500;
+    padding:em(2) em(10) em(2) 0;
+    border-right: 1px solid $color-pf-white ;
+    input[type=checkbox] {
+      margin: 0;
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss]: Why do we have margin 0 here and padding 0 (line 77) below?

Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss]: code from line 77 is not required. It got duplicated. margin 0 we need to overwrite 5px margin top

**Comment by nimishamukherjee** [comment]
[test]

**Comment by nimishamukherjee** [comment]
[test]

File: src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
   cursor: pointer;
   font-size: rem(12);
   align-items: center;
   .row-cbh {
     margin-right: em(10);
     margin-left: em(5);
-    padding-right: em(10);
-    border-right: 1px solid $color-pf-black-500;
+    padding:em(2) em(10) em(2) 0;
+    border-right: 1px solid $color-pf-white ;
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss]: Remove space before ;.

File: src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+      margin: 0;
+      visibility: hidden;
+    }
+    input[type=checkbox]:checked {
+      visibility: visible;
+    }
   }
+  &:hover {
+    .row-cbh {
+       border-right: 1px solid $color-pf-black-500 ;
```
Comment (vikram-raj) [inline src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss]: Remove space before ;.

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 4]
```diff
@@ -364,6 +364,7 @@ class WorkItemListPage {
 
  /* Is checkbox relative to a workitem selected? */
   isWorkItemCheckboxSelected (workItemElement) {
+           browser.actions().mouseMove(workItemElement).perform();
```
Comment (vikram-raj) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Please indent it properly.

**Pull request merged**